layouts/shortcodes/kf-latest-version.html

```diff
@@ -1 +1 @@
-v0.7.1
\ No newline at end of file
+v1.0.0
```

Comment:
We should update this to `v1.0-rc.2` so that the code snippets in the docs tell people where to get the most current RC. Then we just keep changing this for each RC until we have a final v1.0.0.

Reply:
Thanks @sarahmaddox 

I will update it to v1.0-rc.X (I believe there will be a new RC soon.)

Reply:
Updated it to `v1.0-rc.3`